cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-health-check.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-health-check"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-health-check
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-26
+
+== Health checks
+You can intentionally crash your pod making it unresponsive to the Kubernetes liveness probes to see how Kubernetes responds to the event. We will first intentionally crash our pod and see that Kubernetes will self-heal by immediately spinning it back up. Then we will trigger the health check by stopping the response on the `/health` endpoint in our application. After three consecutive failures Kubernetes should stop the pod and then re-create it.
+
+=== Preparing the desktop
+
+. Split your desktop screen between the OpenShift web console and the OSToy application so that you can see the results of your actions immediately.
```

Comment:
```suggestion
. Split your desktop screen between the OpenShift web console and the OSToy application web console so that you can see the results of your actions immediately.
```